internal/resources/resources.go

```diff
@@ -71,9 +76,11 @@ type Resources struct {
 	createdNetNS bool
 	// addedNetNSToVM indicates if the network namespace has been added to the containers utility VM
 	addedNetNSToVM bool
+	// NetworkSetup interface is used to setup and teardown a uvms networking.
+	uvm.NetworkSetup
```

Comment:
I feel like we should have a named field here even if it is just an interface

Reply:
Ended up taking this off the resources struct anyways 😆